localization/react-intl/src/app/components/media/BulkActionsRemoveTag.json

```diff
@@ -32,7 +32,7 @@
   {
     "id": "bulkActionsRemoveTag.dialogBody",
     "description": "Body of dialog warning that bulk untagging items",
-    "defaultMessage": "The <strong>{tags}</strong> tag will be removed from selected items that have this tag. if a tag is not found in a selected item, the item will be unaffected.<br /><br /><strong>This action cannot be undone.</strong> Are you sure you want to continue?"
+    "defaultMessage": "The <strong>{tags}</strong> tag will be removed from selected items.<br /><br /><strong>This action cannot be undone.</strong> Are you sure you want to continue?"
```

Comment:
I noticed the {tags} variable can be a single tag or a list of tags. Perhaps it can read "The {tags} tags will be removed..." and even if the variable gets replaced by a single tag name it still makes sense as it refers to all instances of it on selected items?

Reply:
good idea, done....who are you? the new AO? 😁